### DISCUSSION TITLE
Surcharge

### DISCUSSION THREAD

**Discussion by alphabravocharly72** [mampfes/ha_epex_spot#37] [General]
hello,
must I change the surcharge in the config? If yes how i Can find out?
Thx
 alpha

**Comment by mampfes** [comment]
You can change surcharge in the config dialog of the integration:
![image](https://github.com/mampfes/ha_epex_spot/assets/375950/66f03021-b075-4565-af82-072997ca3abf)

**Reply by alphabravocharly72** [reply]
Hi, I know how to change it. I am looking for the value. Why it is 11.93? Is it necessary to modify it?
Thanks for helping
alpha

**Comment by mampfes** [comment]
It depends on your supplier. 11.93ct is the value for [Awattar Germany](https://www.awattar.de/tariffs/hourly) (sum of "Netznutzung" and "Umlagen, Abgaben, Steuern"). If you have a different supplier then you probably have to change it.

Same is true for the other fields.

**Comment by aichingerk** [comment]
Does anyone know the exact values for Austria Awattar ?

**Comment by mampfes** [comment]
If you have the hourly tariff (https://www.awattar.at/tariffs/hourly), then you have to enter the following values:
Percentage Surcharge: 3%
Absolute Surcharge 1,5 ct/kWh
Tax: 20%

Could you help to improve the documentation so that it becomes clearer what to do and how to get the correct values for your setting?

**Reply by aichingerk** [reply]
What exactly do you mean with improve the documentation ? When i can help, sure no problem

**Reply by mampfes** [reply]
There is already a documentation about the net-price sensor and (maybe a little bit hidden) the configuration options. But it wasn't good enough to answer your questions. Therefore it would be very helpful if you can make a proposal to improve the documentation in a way that it answers your questions directly.
